com.unity.uiwidgets/Runtime/engine2/UIWidgetsPanel.cs

```diff
@@ -40,15 +40,60 @@ public interface IUIWidgetsWindow {
 
         UIWidgetsWindowType getWindowType();
     }
-
-    public class ConfigurationSettings {
-        public static Dictionary<IntPtr, bool> _internalShowDebugLog = new Dictionary<IntPtr, bool>();
-    }
-
     public class Configurations {
         
-        public Dictionary<string, TextFont> _internalTextFonts = new Dictionary<string, TextFont>();
-        public bool _showDebugLog;
+        private Dictionary<string, TextFont> _textFonts = new Dictionary<string, TextFont>();
+
+        bool _debugMode = true;
+
+        public bool debugMode {
+            get { return _debugMode; }
+        }
+
+        public void Clear() {
+            _textFonts.Clear();
+            _debugMode = true;
+        }
+        public void AddFont(string family, TextFont font) {
+           _textFonts[key: family] = font;
+        }
+        public object fontsToObject() {
```

Comment:
good idea to move fontsToObject here :-)

it would be the better to make this a getter: 
public object textFonts => { ... }